src/main/java/org/jabref/logic/importer/WebFetchers.java

```diff
@@ -42,6 +36,13 @@
 import org.jabref.model.entry.identifier.DOI;
 import org.jabref.model.entry.identifier.Identifier;
 
+import java.util.Comparator;
+import java.util.HashSet;
```

Comment:
Something is odd with your import order. Did you import the JabRef code style for Intellij?
https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace#using-jabrefs-code-style